composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": "~5.6 || ~7.1",
         "willdurand/negotiation": "~2.3.1",
-        "zendframework/zend-diactoros": "~1.1.0"
+        "zendframework/zend-diactoros": "^1.7"
```

Comment:
It works with 1.1: let's just update this to `^1.1`

Reply:
N3 framework wants "zendframework/zend-diactoros": "^1.7"

Do we have to match that?

Reply:
No: Composer will be able to resolve the dependency to `1.7.0` or higher since `^1.1` is equivalent to `>=1.1.0 && < 2.0.0`.

Reply:
ok will adjust!